app/services/policies_config_params_wrapper.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class PoliciesConfigParamsWrapper
+
+  attr_reader :policies_config_params
+
+  def initialize(policies_config_params)
+    @policies_config_params = policies_config_params
+  end
+
+  def wrap
+    return policies_config_params if json_param?
```

Comment:
You return a String? No need to parse it?

Then it you could leverage the update in the model

Reply:
`policies_config` is being saved in a JSON.. so to make it "smarter" I would have to decode the array to JSON and then send it to a proxy object. so I would say better to keep it this way..